yarn.lock

```diff
@@ -1098,6 +1098,11 @@
 <<<<<<< HEAD
```

Comment:
Can you delete the yarn.lock file and then run `yarn install` again then check in the new yarn.lock